gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-api/src/main/java/io/gravitee/gateway/handlers/api/spring/ApiHandlerConfiguration.java

```diff
@@ -152,7 +152,8 @@ public ReactorHandlerFactory<Api> reactorHandlerFactory(
         OrganizationManager organizationManager,
         PolicyChainProviderLoader policyChainProviderLoader,
         ApiProcessorChainFactory apiProcessorChainFactory,
-        FlowResolverFactory flowResolverFactory
+        FlowResolverFactory flowResolverFactory,
+        @Qualifier("httpRequestTimeout") Long httpRequestTimeout
```

Comment:
I don't think the qualifier is required as the name is the same as the qualifier value, isn't?

Reply:
you're right

Reply:
done in https://github.com/gravitee-io/gravitee-api-management/pull/2224/commits/10c718e2cb8c3dcfb498664de8c2d42f9e3271f7